packages/front-end/pages/features/[fid].tsx

```diff
@@ -166,7 +172,9 @@ export default function FeaturePage() {
     }
 
     // If there's an active draft, show that by default, otherwise show the live version
-    const draft = data.revisions.find((r) => r.status === "draft");
+    const draft = data.revisions.find(
+      (r) => r.status === "draft" || r.status === "approved"
```

Comment:
This causes some weird behavior.   Like if I request a review, and someone goes to the feature page, it will default to the live revision and it's not obvious that there's any action required.  Same if changes are requested